packages/marko-web-native-x/routes/story.js

```diff
@@ -0,0 +1,25 @@
+const { asyncRoute } = require('@parameter1/base-cms-utils');
+const { getAsObject } = require('@parameter1/base-cms-object-path');
+const createClient = require('../utils/create-client');
+const defaultTemplate = require('../templates/story');
+const defaultFragment = require('../graphql/fragments/story');
+const buildQuery = require('../graphql/queries/story');
+
+module.exports = ({
+  config,
+  template = defaultTemplate,
+  queryFragment = defaultFragment,
+} = {}) => asyncRoute(async (req, res, next) => {
+  if (config.isEnabled()) {
```

Comment:
No need for an enable check... if NX isn't enabled, the site implementor shouldn't include the middleware to begin with